conans/test/utils/tools.py

```diff
@@ -774,6 +774,10 @@ def assert_listed_binary(self, requires, build=False, test=False, test_package=F
             else:
                 raise AssertionError(f"Cant find {r}-{kind} in {reqs}")
 
+    def created_build_folder(self, ref):
```

Comment:
Not really used in the PR?

Reply:
Oops leftover from previous iteration, good catch